pkg/models/alert.go

```diff
@@ -132,9 +138,15 @@ type SaveAlertsCommand struct {
 }
 
 type PauseAlertCommand struct {
-	OrgId   int64
-	AlertId int64
-	Paused  bool
+	OrgId       int64
+	AlertIds    []int64
```

Comment:
ah, I see. And I guess that is potentially something we want, so we can keep that.